astroquery/alma/core.py

```diff
@@ -16,7 +16,7 @@
 from six.moves.urllib_parse import urljoin, urlparse
 from six import iteritems, StringIO
 import six
-from astropy.table import Table, Column
+from astropy.table import Table, Column, vstack as table_vstack
```

Comment:
why do you call it `table_vstack`? we use numpy vstack at one place, and even there we call it `np.vstack`

Reply:
I don't know.   But I think it's useful to keep them distinct

Reply:
thus I vote for keeping the numpy one as np.vstack, and this one vstack, but I agree, this is just bike shedding where there are bigger issues laying around :)